src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
So I tried your suggestion but the chart then doesn't display in a way where the line is more centralized in the graph.  

Using `d[axis]*1.001`
![screen shot 2016-02-10 at 1 32 29 pm](https://cloud.githubusercontent.com/assets/6463914/12960898/f501db3a-cffa-11e5-8b09-3d2d1ebdc99d.png)


Using `Math.ceil(d[axis] / 1000) * 1000`
![screen shot 2016-02-10 at 1 32 52 pm](https://cloud.githubusercontent.com/assets/6463914/12960913/034464ce-cffb-11e5-8d16-5bc87114c853.png)


I should note that I found the `Math.ceil(d[axis] / 1000) * 1000` equation from doing research on D3 chart scaling.  It seems to be a pretty normal approach to padding space on the chart to get your line to be more centralized.